keyform/templates/keyform/add_form.html

```diff
@@ -14,97 +14,69 @@
-                                {{ form.payment_method.label_tag }}
-                                {{ form.payment_method }}
-                            </td>
-                        </tr>
-                    </table>
-                    <table>
-                        <tr>
-                            <td>
-                                {{ form.amt_received.errors }}
-                                {{ form.amt_received.label_tag }}
-                            </td>
-                            <td>
-                                {{ form.amt_received }}
-                            </td>
-                        </tr>
-                        <tr>
-                            <td>
-                                {{ form.charge_amount.errors }}
-                                {{ form.charge_amount.label_tag }}
-                            </td>
-                            <td>
-                                {{ form.charge_amount }}
-                            </td>
-                        </tr>
-                        <tr>
-                            <td>
-                                {{ form.charged_on_rcr.errors }}
-                                {{ form.charged_on_rcr.label_tag }}
-                            </td>
-                            <td>
-                                {{ form.charged_on_rcr }}
-                            </td>
-                        </tr>
-                        <tr>
-                            <td>
-                                <span class="section-header">{% trans 'Comments:' %}</span>
-                            </td>
-                            <td>
-                                <textarea name="comment_text">{{ comment_text }}</textarea>
-                            </td>
-                        </tr>
-                    </table>
+        <div class="row field-wrapper">
+            <div class="large-6 columns">
+                {{ form.request_formset.management_form }}
+                {{ form.building.errors }}
+                {{ form.building.label_tag }}
+                <span class="help-text">{{ form.building.help_text }}</span>
+                {{ form.building }}
+            </div>
+            <div class="large-6 columns">
+                {{ form.bpn.errors }}
+                {{ form.bpn.label_tag }}
+                <span class="help-text">{{ form.bpn.help_text }}</span>
+                {{ form.bpn }}
+            </div>
+            <div class="large-12 columns">
+                {{ form.student_name.errors }}
+                {{ form.student_name.label_tag }}
+                <span class="help-text">{{ form.student_name.help_text }}</span>
+                {{ form.student_name }}
+            </div>
+            <div id="form-set">
+                {% for keydata_form in form.request_formset.forms %}
+                    {% include 'keyform/includes/keydata_form.html' with include_buttons=True %}

```

Comment:
`include_buttons` no longer exists in the included template, so it's no longer necessary here.